osquery/tables/system/cpuid.cpp

```diff
@@ -45,10 +45,14 @@ std::map<int, std::vector<FeatureDef_t> > kCPUFeatures = {
      }},
 };
 
-static inline void cpuid(unsigned int eax, unsigned int ecx, int regs[4]) {
+static inline void cpuid(size_t eax, size_t ecx, int regs[4]) {
+#if defined(WIN32)
+  __cpuidex((int*)regs, (int)eax, (int)ecx);
```

Comment:
Is `cpuid.cpp` ready yet? `osquery_distributed` appears to require it, which is why I included a copy from your `win10-build-test` branch.